### ISSUE TITLE
log4j2 is misleading

### ISSUE THREAD

**Issue by grobmeier** [pavlobaron/log4j2redis#4]
log4j2 is short for log4j 2.0:
logging.apache.org/log4j/2.x

as it seems your appender is targeting log4j 1.x the name might be misleading.

Suggestion: update the readme file and point out for which version of log4j this appender will work.

**Comment by leandrosilva** [comment]
No, it stands for Log4J To Redis, which means logs generated by Log4J is going to be stored on Redis.

**Comment by grobmeier** [comment]
Sure. But which version of log4j? log4j1.x or log4j2.x?
You can only find out that information by looking into the source code.

On Sun, Jan 6, 2013 at 3:14 PM, Leandro Silva notifications@github.comwrote:

> No, it stands for Log4J To Redis, which means logs generated by Log4J is
> going to be stored on Redis.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/pavlobaron/log4j2redis/issues/4#issuecomment-11928893.

## 

http://www.grobmeier.de
https://www.timeandbill.de

**Comment by leandrosilva** [comment]
Oh, I got it now! :) So feel free to fork it and add that. Thanks.
